components/ntask/org.wso2.carbon.ntask.core/src/main/java/org/wso2/carbon/ntask/core/impl/RoundRobinTaskLocationResolver.java

```diff
@@ -39,12 +37,8 @@ public void init(Map<String, String> properties) throws TaskException {
     
     @Override
     public int getLocation(TaskServiceContext ctx, TaskInfo taskInfo) throws TaskException {
-        HazelcastInstance hz = TasksDSComponent.getHazelcastInstance();
-        if (hz == null) {
-            /* this cannot happen, because the task location resolvers are used in clustered mode */
-            return 0;
-        }
-        return (int) Math.abs(hz.getAtomicLong(ROUND_ROBIN_TASK_RESOLVER_ID + ctx.getTaskType()).incrementAndGet());
+        // removed hazelcast
```

Comment:
Add that comment to show that this method has become an empty method which returns 0 because of that. Since we didn't remove the method completely. Should I remove the comment?

Reply:
IMO, it is better if we can remove this comment. However, having an empty method seems redundant to start with (I assume this was unavoidable), if we can add a comment explaining why this is an empty method it would make more sense I guess. For instance, 
```
TODO: Converting to an empty method with the Hazelcast removal effort. Need to revisit the codebase and remove this empty method later.
```
@tharikaGitHub WDYT?

Reply:
I assume this was unavoidable >

Yes, removing these methods may need some senior expertise too (There are number of other methods using these and to remove them, we may need to know which modules and methods we are currently using in our product. For ex: Removed HA and Distributed from carbon-analytics-commons because got informed that our product is not using them now) Also I wasn't able to remove ntask because I was informed that we can't remove that module but had to removed Hazelcast because 4.2 alpha did't work properly because Hazelcast has been already removed from kernel side.

Reply:
@ashera96 will change the comments as requested in a new PR.